tests/testthat/test-pkg-dplyr.R

```diff
@@ -225,3 +225,129 @@ test_that("relocate() for substrait_compiler reorders columns", {
     substrait_select(compiler, col2, col1)
   )
 })
+
+test_that("all types of dplyr mutating joins can be created from substrait_comiler", {
```

Comment:
```suggestion
test_that("all types of dplyr mutating joins can be created from substrait_compiler", {
```
Also, this test and the next one have identical names